.github/workflows/test-build.yaml

```
@@ -24,4 +24,6 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements/${{ matrix.requirements-file }}.txt
-          pip install .
\ No newline at end of file
+          pip install .
+      - name: Test import
+        run: python -c "import sgm"
```

Since the crash is a segfault, you might want to enable [Python development mode](https://docs.python.org/3.11/library/devmode.html#devmode) here to get a chance at a segfault traceback (via `faulthandler`):
```suggestion
        run: python -Xdev -c "import sgm"
```